### DISCUSSION TITLE
Opini

### DISCUSSION THREAD

**Discussion by YoMasihPemula** [deaafrizal/forum#35] [All About Hardware ( PC, Laptop etc...)]
masbro, menurutmu " Lenovo Thinkpad x270 i5 8/254 " itu gmn? dari segi pemakaian gmn, awet brp tahun

**Comment by yohanesokta** [comment]
Menurut ku kalau Lenovo Thinkpad memang tergolong kuat , yang penting optimasi os nya pake win 10 dan minimalkan bloatware, atau pakai linux sekalian

**Reply by YoMasihPemula** [reply]
siap boskuh!!! makaasih banyak

**Comment by shamhasan** [comment]
kalau buat belajar pemrograman saja masih amann mas ku

**Reply by YoMasihPemula** [reply]
siap boskuhh tysm!!!
